src/web/modules/transactions/update/update.http.ts

```diff
@@ -50,7 +50,7 @@ const uploadToS3 = async function uploadToS3(content: string, user: any): Promis
     const key = (user && user.username || '') + moment().format('x')
     logger.info(`Uploading transactions file to S3 with key ${key}`)
     const response = await s3.putObject({
-      Bucket: aws.AWS_S3_UPDATE_TRANSACTIONS_BUCKET_NAME,
+      Bucket: aws.AWS_S3_UPDATE_TRANSACTIONS_V2_BUCKET_NAME,
```

Comment:
Maybe we could call this env var `AWS_S3_UPDATE_TRANSACTIONS_FARGATE_BUCKET_NAME`? As you've probably noticed by now I am reluctant to put `v2` into actual code...

Reply:
I guess so. I kinda wish we had them named the same in the repos. It means the names of deployed things really don't match the names of the things that define their deployments.....but we're already there and at least it's consistent at the moment.

Reply:
Updated https://github.com/alphagov/pay-infra/pull/3249 with the new variable names, will update this now too

Reply:
Sorted